readme.txt

```diff
@@ -3,7 +3,7 @@ Author URI: http://pippinsplugins.com
 Author: Pippin Williamson
 Contributors: mordauk, mindctrl, nosegraze
 Donate link: http://pippinsplugins.com/support-the-site
-Tags: Restrict content, member, members, membership, memberships, member only, registered, registration form, login form, logged in, restricted access, restrict access, limiit access, read-only, read only, access control
+Tags: Restrict content, member, members, membership, memberships, member only, registration form, login form, restricted access, limit access, read only, access control
```

Comment:
There are 13 here still. 12 is max per .org guidelines.

Reply:
@pippinsplugins I count 12.

Reply:
@mindctrl I count 13 when I include the imaginary one in my head